packages/overlay/src/global.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module '@popperjs/core/lib/popper-lite' {
```

Comment:
The new version of popper comes with Typescript definitions, but they are only included in the fully built version of the library. When you use the uncompiled `lib` version, you have to import them manually.

Reply:
I logged https://github.com/popperjs/popper.js/issues/934 for this issue